orchestrator/src/main/java/com/scottlogic/datahelix/generator/orchestrator/generate/GenerateCommandLine.java

```diff
@@ -65,6 +66,12 @@ private void checkForAlphaGenerationDataTypes() {
         }
     }
 
+    @CommandLine.Option(
+        names = { "-V", "--version" },
+        versionHelp = true,
+        description = "Print version information and exit.")
```

Comment:
Perhaps this should be "Prints version information"?

I would imagine that most users would assume that the helix would quit after running this command

Reply:
I wouldn't assume what most users would assume, especially if they combine `--version` with other command-line arguments.

GNU Coreutils specifies that `--version` quits.  GCC doesn't.  The JDK specifies that `-version` quits, but also has a separate `-showversion` switch that doesn't make it quit.

I don't see any need to change what we have here.